pages/common/gh-issue.md

```diff
@@ -5,28 +5,28 @@
 
 - Display a specific issue:
 
-`gh issue view {{issue_number}}`
+`gh issue view {{1..n}}`
 
 - Display a specific issue in the default web browser:
 
-`gh issue view {{issue_number}} --web`
+`gh issue view {{1..n}} --web`
 
 - Create a new issue in the default web browser:
 
 `gh issue create --web`
 
-- List the last 10 issues with the `bug` label:
+- List the last issues with a specific label:
 
-`gh issue list --limit {{10}} --label "{{bug}}"`
+`gh issue list --limit {{1..n}} --label "{{string}}"`
```

Comment:
Again, I don't see anything wrong with specific examples here? Our `{{token_syntax}}` indicates you can change it, and it's quite intuitive as it is....?

Reply:
> Again, I don't see anything wrong with specific examples here?

Range syntax must be used for placeholders where constrained number expected.